libs/state/src/lib/rx-state.service.ts

```diff
@@ -79,6 +72,22 @@ export class RxState<T extends object> implements OnDestroy, Subscribable<T> {
     this.subscription.unsubscribe();
   }
 
+  /**
+   * @description
+   *
+   * Allows to customize state accumulation function.
+   * This can be helpful to implement deep updates and tackle other immutability problems in a custom way.
+   * @example
+   *
+   * ```typescript
+   * const myAccumulator = (state: MyState, slice: Partial<MyState>) => ({
+   * ...state,
+   * ...slice
+   * });
```

Comment:
```suggestion
  * const myAccumulator = (state: MyState, slice: Partial<MyState>) => deepCopy(state, slice);
```

Reply:
Example right now is the default behavior. Let's go with something different to not confuse people